packages/esm-outpatient-app/src/patient-search-icon/patient-search-icon.component.scss

```diff
@@ -0,0 +1,69 @@
+@import "~@openmrs/esm-styleguide/src/vars";
+@import "~carbon-components/src/globals/scss/vendor/@carbon/layout/scss/generated/spacing";
+$inverse-02:#393939;
```

Comment:
This is only used once. Just inline it. If you believe it is missing from [esm-styleguide](https://github.com/openmrs/openmrs-esm-core/blob/master/packages/framework/esm-styleguide/src/_vars.scss), please add it there.